zero_bin/common/src/prover_state/persistence.rs

```diff
@@ -270,6 +270,21 @@ pub fn persist_all_to_disk(
     Ok(())
 }
 
+/// Flushes all existing prover state configurations and associated circuits
+/// that have been written to disk.
+pub fn delete_all() -> anyhow::Result<()> {
+    let circuit_dir = circuit_dir();
+    let path = Path::new(&circuit_dir);
+
+    if path.is_dir() {
+        // We will delete the entire directory and recreate it after.
+        fs::remove_dir_all(path)?;
```

Comment:
Hmm fair enough, but then we'd need to define what are these checks.

We don't have any file extension for these files, so what we could check is if they start with "prover_state" / "verifier_state", this should be enough?

Reply:
> We don't have any file extension for these files, so what we could check is if they start with "prover_state" / "verifier_state", this should be enough?

Could we also make the convention for the circuits dir name to always finish with `circuits` like `my_circuits`, `production_test_plonky2_circuits` and error if this is not satisfied? 
And not use current directory for circuits but make `circuits` sub-directory in `set_circuit_cache_dir_env_if_not_set`.

Reply:
> And not use current directory for circuits but make circuits sub-directory in set_circuit_cache_dir_env_if_not_set.

If the former is satisfied (i.e. we enforce the directory must end in `_circuits`, then we probably don't need this constraint?

Reply:
(I've addressed the other two concerns, i.e. specific files wiping and naming convention check, remains the sub-directory thing that I can add if you want to).

Reply:
Lol nice catch @atanmarko!

Actually I think maybe instead of falling back to the `cwd` maybe `set_circuit_cache_dir_env_if_not_set` should instead set the circuits directory to `<cwd>/circuits` instead, which would also cover this case. Also feels like just dumping circuits in the `cwd` is a bit messy.